src/App.jsx

```diff
@@ -86,11 +86,10 @@ export default function ReplicadApp() {
             activeAtom.setAlert("Can't display Mesh " + e);
           });
         /*Set wireMesh*/
-
         //Exception: Don't display the mesh if the thing we are displaying is already the output
-        if (GlobalVariables.currentMolecule.output.value != id) {
+        if (GlobalVariables.currentMolecule.uniqueID != id) {
```

Comment:
This is exactly what I was thinking that it should be, I couldn't figure out why it was value. As far as I can tell output.value was never used anywhere 🤷‍♂️